locust/static/chart.js

```diff
@@ -31,6 +31,16 @@
                     x: 10,
                     y: 10,
                 },
+                toolbox:{
+                	show:true,
+                	feature:{
+                		saveAsImage:{
```

Comment:
does this just give you an image of the graph? If so, it serves a different purpose than #693 which actually gives you all of the data points in a CSV file

Reply:
yes。only the graph。